### PR TITLE
chore: temporarily remove cn-* regions due to ci instability (socket timed out without establishing a connection)

### LINKED ISSUE #765: cli: list/destroy: AWS region lookup may fail for cn-north-1 / cn-northwest-1 with TCP connect timeouts
prior discussion: https://github.com/opstrace/opstrace/issues/511#issuecomment-803967724

> Educated speculation: based on past experience, this happens when AWS API endpoints in and around Asia are not TCP-reachable.

> Do not support all AWS regions. The ones that make problems here are often in Asia. Of course we also want users in Asia. And not supporting those has a weird taste -- feels like discrimination.

```
[2021-05-20T13:16:05Z] 2021-05-20T13:16:05.799Z info: Found Opstrace clusters in AWS us-west-2: prs-bk-4266-4a2-a, prs-bk-4267-b87-a, tracy, prs-bk-4265-a60-a, prs-bk-4268-5d3-a, schedul-bk-2320-826-a, schedul-bk-2321-5c5-a, upgrade-bk-632-703-a, chris6, jpload-1621421101, matdemo, prs-bk-4245-d39-a
[2021-05-20T13:16:09Z] 2021-05-20T13:16:09.750Z info: AWS API request failed (attempt 0): Socket timed out without establishing a connection (retry in 1.55 seconds)
[2021-05-20T13:16:09Z] 2021-05-20T13:16:09.750Z info: AWS API request failed (attempt 0): Socket timed out without establishing a connection (retry in 1.07 seconds)
[2021-05-20T13:16:16Z] 2021-05-20T13:16:16.830Z info: AWS API request failed (attempt 1): Socket timed out without establishing a connection (retry in 2.99 seconds)
[2021-05-20T13:16:17Z] 2021-05-20T13:16:17.302Z info: AWS API request failed (attempt 1): Socket timed out without establishing a connection (retry in 1.81 seconds)
[2021-05-20T13:16:25Z] 2021-05-20T13:16:25.124Z info: AWS API request failed (attempt 2): Socket timed out without establishing a connection (retry in 3.21 seconds)
[2021-05-20T13:16:25Z] 2021-05-20T13:16:25.824Z info: AWS API request failed (attempt 2): Socket timed out without establishing a connection (retry in 3.22 seconds)
[2021-05-20T13:16:34Z] 2021-05-20T13:16:34.340Z info: AWS API request failed (attempt 3): Socket timed out without establishing a connection (retry in 7.87 seconds)
[2021-05-20T13:16:35Z] 2021-05-20T13:16:35.044Z info: AWS API request failed (attempt 3): Socket timed out without establishing a connection (retry in 7.01 seconds)
[2021-05-20T13:16:48Z] 2021-05-20T13:16:48.061Z info: AWS API request failed (attempt 4): Socket timed out without establishing a connection (retry in 11.31 seconds)
[2021-05-20T13:16:48Z] 2021-05-20T13:16:48.211Z info: AWS API request failed (attempt 4): Socket timed out without establishing a connection (retry in 13.00 seconds)
[2021-05-20T13:17:05Z] 2021-05-20T13:17:05.383Z info: AWS API request failed (attempt 5): Socket timed out without establishing a connection (retry in 20.02 seconds)
[2021-05-20T13:17:07Z] 2021-05-20T13:17:07.215Z info: AWS API request failed (attempt 5): Socket timed out without establishing a connection (retry in 11.67 seconds)
[2021-05-20T13:17:24Z] 2021-05-20T13:17:24.897Z info: AWS API request failed (attempt 6): Socket timed out without establishing a connection (retry in 19.15 seconds)
[2021-05-20T13:17:31Z] 2021-05-20T13:17:31.410Z info: AWS API request failed (attempt 6): Socket timed out without establishing a connection (retry in 25.81 seconds)
[2021-05-20T13:17:50Z] 2021-05-20T13:17:50.056Z info: AWS API request failed (attempt 7): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:18:03Z] 2021-05-20T13:18:03.234Z info: AWS API request failed (attempt 7): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:18:26Z] 2021-05-20T13:18:26.077Z info: AWS API request failed (attempt 8): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:18:39Z] 2021-05-20T13:18:39.247Z info: AWS API request failed (attempt 8): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:19:02Z] 2021-05-20T13:19:02.096Z info: AWS API request failed (attempt 9): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:19:15Z] 2021-05-20T13:19:15.255Z info: AWS API request failed (attempt 9): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:19:38Z] 2021-05-20T13:19:38.113Z info: AWS API request failed (attempt 10): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:19:51Z] 2021-05-20T13:19:51.265Z info: AWS API request failed (attempt 10): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:20:14Z] 2021-05-20T13:20:14.134Z info: AWS API request failed (attempt 11): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:20:27Z] 2021-05-20T13:20:27.273Z info: AWS API request failed (attempt 11): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:20:50Z] 2021-05-20T13:20:50.153Z info: AWS API request failed (attempt 12): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:21:03Z] 2021-05-20T13:21:03.285Z info: AWS API request failed (attempt 12): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:21:26Z] 2021-05-20T13:21:26.177Z info: AWS API request failed (attempt 13): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:21:39Z] 2021-05-20T13:21:39.301Z info: AWS API request failed (attempt 13): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:22:02Z] 2021-05-20T13:22:02.202Z info: AWS API request failed (attempt 14): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:22:15Z] 2021-05-20T13:22:15.309Z info: AWS API request failed (attempt 14): Socket timed out without establishing a connection (retry in 30.00 seconds)
[2021-05-20T13:22:38Z] 2021-05-20T13:22:38.221Z debug: aws sdk: [AWS eks undefined 394.478s 15 retries] listClusters({})
[2021-05-20T13:22:38Z] 2021-05-20T13:22:38.222Z debug: this does not appear to be an AWS API error
[2021-05-20T13:22:38Z] 2021-05-20T13:22:38.222Z error: eks.listClusters() for region cn-north-1 failed unexpectedly with error: Socket timed out without establishing a connection
[2021-05-20T13:22:38Z] 2021-05-20T13:22:38.222Z error: Could not look up clusters in region cn-north-1.
[2021-05-20T13:22:38Z] 2021-05-20T13:22:38.223Z debug: shut down logger, then exit with code 1
```

### PR DESCRIPTION
**Pull request by triclambert**
Resolves #765.

### CODE REVIEW

**Comment by triclambert** [comment]
AWS passed; GCP tests failed as of known issue.  Merging.

File: packages/cli/src/list.ts [hunk 1]
```diff
@@ -107,9 +107,7 @@ export async function EKSgetOpstraceClustersAcrossManyRegions(): Promise<
     "us-east-1",
     "us-east-2",
     "us-west-1",
-    "us-west-2",
-    "cn-north-1",
-    "cn-northwest-1"
+    "us-west-2"
```
Comment (jgehrcke) [inline packages/cli/src/list.ts:110]: thanks for that diff, too :)

**Comment by jgehrcke** [comment]
Thanks! Just want to say: the change is easy. The discussion around this change is the challenging part. I am just a little surprised that there is no discussion here on this pull request, no context, no statement for how to move forward. Let's add this post-merge.

How do we think about "temporarily?" here in the title? I think if we're honest it means "for now" which means until someone needs that. And if we think "yeah of course that's the way to go" then my reply would be: that's exactly the kind of thing to put into the pull request, commit message, or code comment.

The list itself certainly requires a code comment pointing to the relevant tickets.

Also also need to prepare a changelog entry for the next release.

I was hesitant to "just do this" before because of the idea of explaining this to the world, also see my quote from https://github.com/opstrace/opstrace/issues/765#issue-896852600:

> Do not support all AWS regions. The ones that make problems here are often in Asia. Of course we also want users in Asia. And not supporting those has a weird taste.

**Comment by jgehrcke** [comment]
Also, the CI failures need comments upon merge.

**Comment by triclambert** [comment]
> would be great to link to the relevant issues or ideally specific comments in issues.

Done.

> How do we think about "temporarily?" here in the title? I think if we're honest it means we will forget about this until someone needs that. 

To make my intent explicit:  "disable cn-* regions until someone needs them." 

I will open a separate issue to discuss adding support for them.

**Pull request merged**